app/views/layouts/_logged_in_header.html.haml

```diff
@@ -0,0 +1,5 @@
+%header.sticky.top-0.left-0.right-0.flex.justify-between.items-center.py-2.px-3.max-w-dvw.overflow-scroll.h-8.z-100.border-b-1.border-neutral-800.text-base{class: 'text-(--main-text-color,var(--text-color)) bg-(--main-bg-color,var(--background-color))'}
```

Comment:
`--main-bg-color` is set in `Home.vue` and `Logs.vue`.

`--main-text-color` is set in `Home.vue`.

`--text-color` and `--background-color` are set by the blog.